starky/src/proof.rs

```diff
@@ -33,7 +33,7 @@ pub struct StarkProof<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>,
 
 impl<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const D: usize> StarkProof<F, C, D> {
     /// Recover the length of the trace from a STARK proof and a STARK config.
-    pub(crate) fn recover_degree_bits(&self, config: &StarkConfig) -> usize {
+    pub fn recover_degree_bits(&self, config: &StarkConfig) -> usize {
```

Comment:
Maybe we can also make the `recover_degree_bits` function on `StarkProofTarget` public as well.

Reply:
good idea!